README.md

```diff
@@ -260,22 +260,22 @@ build:
         current: String | List of Strings
         upstream: String | List of Strings
         downstream: String | List of Strings
-        merge: String[]
       current: String | List of Strings
       upstream: String | List of Strings
       downstream: String | List of Strings
       after:
         current: String | List of Strings
         upstream: String | List of Strings
         downstream: String | List of Strings
-        merge: String[]
-      merge: String[]
```

Comment:
so I just notice `merge` mechanism is not supported here? :thinking: If this is the case we should open new issue for it...

Reply:
yeah unfortunately i missed that. Will open another issue

Reply:
@shubhbapna please link here

Reply:
https://github.com/kiegroup/build-chain-configuration-reader/issues/71